{"_id":"91d99160-f7e0-33f1-b51a-873f1fff3ecf","authorName":"Zarat","authorEmailEncrypted":"RpL2VEryMBQ7pXHV5DGE83RtxFLAKODskk/ylnheNxj540STNYgNYj1l6nOTCA8+yumYupwDVqdbb/h9Wh/75PP6V4RUE5e/mtWfhTQBFL7HAZ7G0KA88IB/Xq2th9cYCpvsVt1Pe7mP9+4cnfRtkCSpaeGIJQ7v/1ypD/X1dLeK/9MYa7cSqIe2cW3Ke35KrYlbSMh+887+C3wKyPN5u6yfOljNA5LlSZDPBUt+RmCXEifiCIXBxoetGr650gNmQf8WE/Qez0Ev5A3xcnosO2srYFKvlxebnRphaFd/GcaoK6mTyZyZqWMHbjMo4XUyfsO9mb+FPCdMThRvGQ/kZZ3Xn4TjdYwU1IMD5DQcvz+/fpTzdPhM5do376xIOi9xeipcypHCaqIHbnajgj9YSbWL2W5jCKqXTy9S6/Ne/kW3GLbEeQZu6Qr96GxfvbvPRhOTQVvmOUNvwGSJdzqMKDLEmzpL4gg+iIVPtdhqGrjttcugFiR0kcuMZGkdMlBZXxjqJ3PMiOifvEeQyso85xa0pcN4OnW2eyjAQNO5GJI+CXuxlN8pLA0ikxpN3iUJKq5Mc6dTHYKHsMuXiDRVligRG9zuHlTLIuUWw+LjqLhPHdB+oXbX3bt0jTT0bpaubwufEGOdXngSWSHGUy+ZK6QX172g0QyNiGbr2zwrhkA=","authorEmailMD5":"d307e42a92fbce947bfa9fae4400e95f","message":"If you're on 4.5 you can use System.Runtime.ExceptionServices.ExceptionDispatchInfo to store the exception and rethrow it without overwriting the stack trace.","postId":"2014_06_await-in-catch-and-finally-196df97f-fc37-3a25-b9e6-1b95025b3836","replyTo":"e0dc4f0e-ae83-39b8-9ce0-b23ee457470f","date":"2014-06-16T08:23:07Z","timestamp":1402906987,"authorUserId":"disqus:disqus_bCQ4ScqAv7"}